qa/L1_FW_iterators_separate_executor/test.h

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash -e
+# used pip packages
+pip_packages="tensorflow-gpu torchvision mxnet-cu##CUDA_VERSION##"
+
+pushd ../..
+
+cd dali/test/python
+
+NUM_GPUS=$(nvidia-smi -L | wc -l)
+
+test_body() {
+    python test_RN50_data_fw_iterators.py --gpus ${NUM_GPUS} -b 13 --workers 3 --separate_queue --cpu_size 2 --gpu_size 2 --epochs 3
```

Comment:
After discussing with Janusz. This test takes quite a long time to run (~1h) so it should be L2. As an alternative you could limit the number of iterations `-i 100`

Reply:
I think a few iterations will be sufficient here and let us keep it as a L1 test